generate_static_assets.ts

```diff
@@ -77,8 +77,10 @@ async function createStaticAssetFromPath(
 ): Promise<File> {
   logger.debug("Reading", path);
   const bytes = await Deno.readFile(path);
+  const webkitRelativePath = relative(root, path);
   return Object.assign(new Blob([bytes]), {
-    name: join(distPrefix, relative(root, path)),
+    name: join(distPrefix, webkitRelativePath),
     lastModified: 0,
+    webkitRelativePath,
```

Comment:
Why is this necessary?

Reply:
![image](https://github.com/kt3k/packup/assets/64361584/197bbcc8-27f4-4bc6-a33e-c2d4814b61fa)
It said `File` needs to has [webkitRelativePath](https://developer.mozilla.org/en-US/docs/Web/API/File/webkitRelativePath).

Reply:
Though, do you think that it is better to just use `File` constructor? Since you want it to return `File` object so, will you let me refactor all of the `Object.assign` + `new Blob`?

Reply:
I think it's fine for now. Thanks for updating!

Reply:
Though I already did it in my local, mind if I make another pull request regarding this?

Reply:
BTW while I'm fine with `webkitRelativePath` (it's at least harmless), I'm curious where that type error comes from because Deno and TypeScript don't seem having that property defined in File. https://github.com/search?q=repo%3Adenoland%2Fdeno%20webkitRelativePath&type=code

Reply:
> Though I already did it in my local, mind if I make another pull request regarding this?

Yes, that's also welcome!

Reply:
> 

[lib.dom.ts](https://github.com/denoland/deno/blob/0c3bbf7acdeacdd1c36b4b847679d85ca19962dc/cli/tsc/dts/lib.dom.d.ts#L8220)